.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    # run CI every day even if no PRs/merges occur
+    - cron:  '0 6 * * *'
+jobs:
+  Native_Linux:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Native LLVM 8.0 on Ubuntu 18.04 Build
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y ninja-build unzip libomp-dev libomp5 libtinfo-dev lsb-release
+        scripts/build.sh --llvm-version llvm800 --prefix /opt/trailofbits/rellic --use-host-compiler --extra-cmake-args "-DCMAKE_BUILD_TYPE=Release"
+        cd rellic-build
+        CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --verbose --target test
+        CTEST_OUTPUT_ON_FAILURE=1 cmake --build . --target install
+        cd ..
+    - name: Test Rellic Build
+      run: |
+        scripts/roundtrip.py /opt/trailofbits/rellic/bin/rellic-decomp-8.0 $(realpath tests/tools/decomp) $(which clang-8)
+  Docker_Linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        llvm: ["800"]
```

Comment:
Do we want to test more than just LLVM 800?

Reply:
We do when they are supported :)

Right now Rellic only builds against LLVM8 due to LLVM API differences